gui/src/patient_detached_alarm.h

```diff
@@ -27,12 +27,9 @@ class PatientDetachedAlarm : public LatchingAlarm {
     VolumetricFlow flow_correction =
         ml_per_min(status.sensor_readings.flow_correction_ml_per_min);
     if (flow_correction <= threshold_) {
-      return {
-          QString(
-              "Patient detached from device: measured flow correction of %1 "
-              "ml/s, threshold %2 ml/s")
-              .arg(QString::number(flow_correction.ml_per_sec(), 'f', 0),
-                   QString::number(threshold_.ml_per_sec(), 'f', 0))};
+      return {QString("Patient detached from device")
+                  .arg(QString::number(flow_correction.ml_per_sec(), 'f', 0),
```

Comment:
Presumably you should get rid of the args?

Reply:
Done.